tests/transformer/test_dict_comprehension.py

```diff
@@ -24,10 +24,23 @@ def test_dict_comprehension_with_attrs(mocker):
     assert ret == {1: 'b'}
 
     _getiter_.assert_called_once_with(z)
-    _getattr_.assert_has_calls([
-        mocker.call(seq, 'z'),
-        mocker.call(z[0], 'k'),
-        mocker.call(z[1], 'k'),
-        mocker.call(z[1], 'v'),
-        mocker.call(z[1], 'k')
-    ])
+
+    calls = [mocker.call(seq, 'z')]
+
+    # Note: Order changed in PEP 572, starting with Python 3.8.
+    if version_info.major == 3 and version_info.minor >= 8:
```

Comment:
We already have a constant for this comparison, see

https://github.com/zopefoundation/RestrictedPython/blob/49fe489c2bd08c6451bdc1e8a3df716edc4faca6/src/RestrictedPython/_compat.py#L12

Reply:
Fixed